patches/master_patch.patch

```diff
@@ -512,6 +566,50 @@ index d85ddf9db90d332a31f839e7600b8d6e5ec61356..2dbe745b19001ce55631e625b6c597b2
    // Finds TabStripModel which has a WebContents whose id is the given
    // |tab_id|, and returns the WebContents index and the TabStripModel.
    int FindTabStripModelById(int32_t tab_id, TabStripModel** model) const;
+diff --git a/chrome/browser/safe_browsing/chrome_password_protection_service.cc b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+index 4275342f9244aa005521882c23767662fa2a681c..de4b9e02182dd4ed34e248d993c2a5f1e480d020 100644
+--- a/chrome/browser/safe_browsing/chrome_password_protection_service.cc
++++ b/chrome/browser/safe_browsing/chrome_password_protection_service.cc
+@@ -672,8 +672,7 @@ bool ChromePasswordProtectionService::UserClickedThroughSBInterstitial(
+ }
+ 
+ AccountInfo ChromePasswordProtectionService::GetAccountInfo() {
+-  SigninManagerBase* signin_manager =
```

Comment:
stub `SigninManagerFactory::GetForProfileIfExists` instead of patching here. See site_engagement_service.cc for an example. You don't have to stub any other `SigninManagerBase` method because they will be stripped